app/models/gift.rb

```diff
@@ -22,14 +22,16 @@ def has_name_or_pseud
     end
   end
 
-  validates_uniqueness_of :pseud_id, scope: [:work_id], message: ts("You can't give a gift to the same person twice.")
+  validates_uniqueness_of :pseud_id,
+    scope: :work_id,
```

Comment:
Is this functionally different ? I can see [:work_id] being different  but what is the change ?